doc/source/whatsnew/v0.24.1.rst

```diff
@@ -66,7 +66,7 @@ Bug Fixes
 **Reshaping**
 
 - Bug in :func:`merge` when merging by index name would sometimes result in an incorrectly numbered index (:issue:`24212`)
-
+Bug in ``DataFrame.groupby(pd.Grouper(freq='1d'))`` when there is a time change (DST) and grouping frequecy is 1d (:issue:`24972`)
```

Comment:
Use ``:meth:`DataFrame.groupby`` with a separate ``:class:Grouper`` and ``freq='1D'``

Reply:
done